docs/pages/content/schemas.tsx

```diff
@@ -11,7 +11,7 @@ const SchemasPage: React.FunctionComponent = () => {
       <h1>Schema Groups</h1>
 
       <p>
-        This is the schemas exported by nimbus-shared. The source of these schemas is Typescript {" "}
+        This is the schemas exported by nimbus-shared. The source of these schemas is Typescript{" "}
```

Comment:
This is my fault, btw. I foolishly pushed to master with "oh it's just a little typo fix".